pkg/controllers/metrics/pod/controller.go

```diff
@@ -36,7 +36,7 @@ import (
 
 	"github.com/aws/karpenter-core/pkg/apis/provisioning/v1alpha5"
 	"github.com/aws/karpenter-core/pkg/metrics"
-	operatorcontroller "github.com/aws/karpenter-core/pkg/operator/controller"
+	corecontroller "github.com/aws/karpenter-core/pkg/operator/controller"
```

Comment:
Why alias?
```suggestion
	"github.com/aws/karpenter-core/pkg/operator/controller"
```

Reply:
Because we are using `controller` package from `controller-runtime` as well